README.md

```diff
@@ -177,46 +125,106 @@ Once the data model is defined, you can [generate Prisma Client](https://www.pri
 
 ### Accessing your database with Prisma Client
 
-#### Generating Prisma Client
+#### Step 1: Install Prisma
 
-The first step when using Prisma Client is installing its npm package:
+First, install Prisma CLI as a development dependency and Prisma Client:
 
 ```
+npm install prisma --save-dev
 npm install @prisma/client
 ```
 
-Note that the installation of this package invokes the `prisma generate` command which reads your Prisma schema and _generates_ the Prisma Client code. The code will be located in `node_modules/.prisma/client`, which is exported by `node_modules/@prisma/client/index.d.ts`.
+#### Step 2: Set up your Prisma schema
 
-After you change your data model, you'll need to manually re-generate Prisma Client to ensure the code inside `node_modules/.prisma/client` gets updated:
+Ensure your Prisma schema includes a `generator` block with an `output` path specified:
+
+```prisma
+generator client {
+  provider = "prisma-client"
+  output   = "./generated"
+}
+
+datasource db {
+  provider = "postgresql"  // mysql, sqlite, sqlserver, mongodb or cockroachdb
+}
+```
+
+#### Step 3: Configure Prisma Config
+
+Configure the Prisma CLI using a `prisma.config.ts` file. This file configures Prisma CLI subcommands like `migrate` and `studio`. Create a `prisma.config.ts` file in your project root:
+
+```ts
+import { defineConfig, env } from "prisma/config";
+
+type Env = {
+  DATABASE_URL: string
+}
+
+export default defineConfig({
+  schema: "prisma/schema.prisma",
+  migrations: {
+    path: "prisma/migrations",
+  },
+  datasource: {
+    url: env<Env>("DATABASE_URL"),
+  },
+});
+```
+
+**Note**: Environment variables from `.env` files are not automatically loaded when using `prisma.config.ts`. You can use `dotenv` by importing `dotenv/config` at the top of your config file, or when using `tsx`, you can pass a `--env-file` flag to load environment variables. For Bun, `.env` files are automatically loaded.
```

Comment:
> when using `tsx`, you can pass a `--env-file` flag to load environment variables

I don't think this would help for `prisma.config.ts`. For an application using Prisma Client, if you want to use env vars there, sure. But you're not running Prisma CLI with tsx, and `prisma.config.ts` is only used by Prisma CLI.